e2e/libs/keywords/recurringjob_keywords.py

```
@@ -36,3 +36,15 @@ def create_backup_recurringjob_for_volume(self, volume_name):
     def check_recurringjobs_work(self, volume_name):
         logging(f'Checking recurringjobs work for volume {volume_name}')
         self.recurringjob.check_jobs_work(volume_name)
+
+    def create_system_backup_recurringjob(self, job_name, parameters={'volume-backup-policy': 'if-not-present'}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)